packages/@okta/vuepress-site/docs/concepts/authentication/index.md

```diff
@@ -7,80 +7,43 @@ meta:
 
 ### Authentication API
 
-The underlying foundation for the Sign-In Widget and Auth SDK is a
-comprehensive [authentication REST API](/docs/reference/api/authn/)
-exposed through Okta.  Use it as a
-standalone API to provide the identity layer on top of your existing
-application and authentication logic, or use it with the Okta [Sessions API](/docs/reference/api/sessions/)
-to obtain an Okta [session cookie](/docs/guides/session-cookie/) and access apps within Okta.
-This session integration provides an SSO experience across custom and Okta-managed apps.
-
-## Building Apps That Support SSO
-
-For IT or ISVs who want to use Okta as an identity provider, Okta
-provides several options for secure SSO.  SAML has been
-widely used as the single sign-on protocol by many ISVs and is
-supported by many identity management solutions.  Okta provides
-comprehensive guidance for developers to implement a proper
-[SAML service provider](https://www.okta.com/integrate/documentation/saml/).
-For IT building internal apps and would like to support SSO, SAML is
-also a good option.
-
-OpenID Connect is the emerging technology that provides an alternative implementation of SSO.
-Okta is a [Certified OpenID Connect provider](http://openid.net/certification/).
-Building on top of OAuth 2.0 framework, OpenID Connect is a modern
-implementation to support authentication and SSO.  If you
-are an Okta customer, our [OpenID Connect API](/docs/reference/api/oidc/) is a great way to support SSO and
-is a simpler alternative to SAML.
-
-For ISVs who want to provide SSO for their customers, both
-SAML and OpenID Connect are worth considering. Both protocols cover a
-wide variety of identity providers that you may encounter.
+The underlying foundation for the Sign-In Widget and Auth SDK is a comprehensive [authentication REST API](/docs/reference/api/authn/) exposed through Okta.  Use it as a standalone API to provide the identity layer on top of your existing application and authentication logic, or use it with the Okta [Sessions API](/docs/reference/api/sessions/) to obtain an Okta [session cookie](/docs/guides/session-cookie/) and access apps within Okta. This session integration provides an SSO experience across custom and Okta-managed apps.
+
+## Building Apps That Support Single Sign-On
+
+For IT developers or ISVs who want to use Okta as an identity provider, Okta provides several options for secure SSO.  SAML has been widely used as the single sign-on protocol by many ISVs and is supported by many identity management solutions.  Okta provides comprehensive guidance for developers to implement a proper [SAML service provider](https://www.okta.com/integrate/documentation/saml/). For IT developers building internal apps and would like to support SSO, SAML is also a good option.
```

Comment:
...... For IT developers building internal apps that would like to support SSO, SAML is also a good option.

Reply:
Fixed.